Makefile

```diff
@@ -1,18 +1,9 @@
 HERCULE := ./node_modules/.bin/hercule
 DREDD := ./node_modules/.bin/dredd
-SOURCES := apiary.apib sample.apib introduction.md \
-	root.apib parser.apib composer.apib \
-	sample.refract.parse-result.json \
-	sample.refract.parse-result.yaml \
-	sample.apiblueprint.parse-result.json \
-	sample.apiblueprint.parse-result.yaml \
-	sample.apiblueprint.ast-2.0.yaml \
-	sample.apiblueprint.ast-2.0.json
-DEPENDENCIES = $(foreach file,$(SOURCES),source/$(file))
 HOST := https://api.apiblueprint.org/
 APIARY_API := apiblueprintapi
 
-apiary.apib: node_modules $(DEPENDENCIES)
```

Comment:
Why this change? This means if I run `make apiary.apib`, then make a change to one of the included files and run `make apiary.apib` again it won't be updated.